cmd/ecsfargate.go

```diff
@@ -581,6 +579,8 @@ aws_profile = "{{ .AwsProfile }}"
 	
 saml_role = "{{ .AwsSamlRole }}"
 
+secrets_saml_users = []
```

Comment:
unused variable?

Reply:
Without it, TF prompts for it with `secretsmanager.tf`.  Be default, the current user gets added.  Filling out that field lets users add more people.